### PR TITLE
Generate Bill Button added in Transport and Tada

### PR DESCRIPTION
**Pull request by haramrit**

### CODE REVIEW

File: src/librehatti/suspense/views.py [hunk 1]
```diff
@@ -925,4 +926,97 @@ def summary_page(request):
...
+                l1.append(c)
+            d  = str(value['date'])[1:-1]
+            e = d.split(',')
+            for a in e:
+                c = a.replace("u'","")
+                c = c.replace("'","")
+                l.append(c)
+            distance = 0
+            for temp_var in l1:
+                total = rate * int(temp_var)
```
Comment (jasvir99) [inline src/librehatti/suspense/views.py]: Don't use variables like a,b,c.

Comment (haramrit) [inline src/librehatti/suspense/views.py]: On 2 January 2015 at 21:35, jasvir99 <notifications@github.com> wrote:
>
> Don't use variables like a,b,c.

Oops,Sorry missed this point.
Should I cancel the pull request and resend it?


-- 
Har Amrit Pal Singh
http://haramritpanesar.wordpress.com/

Comment (jasvir99) [inline src/librehatti/suspense/views.py]: yes.

**Comment by jasvir99** [comment]
:-1:

**Comment by haramrit** [comment]
On 4 January 2015 at 20:25, jasvir99 <notifications@github.com> wrote:
>
> Closed #363.



Sorry Github is not working.
I tried it with dns 8.8.8.8 and 8.8.4.4 as well
but it didnot work.
I will commit it today at tcc.
Sorry for inconvenience

-- 
Har Amrit Pal Singh
http://haramritpanesar.wordpress.com/

**Pull request closed without merging**